pages/controls/Certificate_and_Public_Key_Pinning.md

```diff
@@ -2,798 +2,127 @@
 
-Ephemeral keys are temporary keys used for one instance of a protocol
-execution and then thrown away. An ephemeral key has the benefit of
-providing forward secrecy, meaning a compromise of the site or service's
-long term (static) signing key does not facilitate decrypting past
-messages because the key was temporary and discarded (once the session
-terminated).
+For the purposes of this document OS and browser trust store can be used interchangeably. Some browsers, like Safari and Edge use the OS’s trust store. Chrome browser and OS have their own store, but when the browser runs on non-ChormeOS, it also trusts the OS store, and Firefox has its own TLS stack and store. In the mobile space, this is similar, as Android uses the Google store and Apple uses their own, both which apply to applications on the platform, not just browsers. 
 
-Ephemeral keys do not affect pinning because the Ephemeral key is
-delivered in a separate `ServerKeyExchange` message. In addition, the
-ephemeral key is a key and not a certificate, so it does not change the
-construction of the certificate chain. That is, the certificate of
-interest will still be located at `certificates[0]`.
+The [CA Browser Forum](https://cabforum.org/) (CA/BF) is another organization working to make sure that the PKI industry is in sync. The Forum is an unincorporated association of separate organizations. It is made up of CA vendors, browser vendors, and other key parties with an interest in securing the internet. This collaboration between both industries is useful in finding common ground when incentives do not align. The Forum has voting members and working groups that come together to evaluate and address risks in the space. They set [baseline requirements](https://cabforum.org/baseline-requirements-documents/) for how CAs are designed and operated. These rules must also be followed for a CA to be accepted into and remain in a Mozilla, Microsoft, Apple, or Google trust store.
```

Comment:
This collaboration between both industries ➡️ This collaboration between industries

The text outlines more then two, hence not "both"